src/components/CheckboxToggle/__tests__/CheckboxToggle.spec.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import CheckboxToggle from '../CheckboxToggle';
+
+const getCmp = (props = {}) => shallow(<CheckboxToggle id="foo" label="bar" {...props} />);
```

Comment:
I thought about this. Maybe I have an abbreviation intolerance ;-) (You use `getComponent` in other specs. But maybe I'm too picky with this).